runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -206,6 +207,11 @@ public void start() throws Exception
 	public void shutdown()
 	{
 		discordService.close();
+
+		if (SystemTray.isSupported())
+		{
+			SystemTray.getSystemTray().remove(gui.getTrayIcon());
```

Comment:
This does not change anything for me on Windows. It removes the tray icon on closing the frame even without this change.

Reply:
Weren't there reports that the tray icon is not being removed? So now it is working?

Reply:
That seems to only happen when I force terminate the process from my IDE.

Reply:
Ah, well, in that case no cleanup logic is possible. So i guess that will be won't fix and I will remove this commit from this PR.